scripts/zones/Leujaoam_Sanctum/mobs/Leujaoam_Worm.lua

```diff
@@ -18,7 +18,7 @@ entity.onMobDeath = function(mob, player, optParams)
 end
 
 entity.onMobDespawn = function(mob)
-    xi.assault.progressInstance(mob)
+    xi.assault.progressInstance(mob, 1)
```

Comment:
The real fix for this since it was changed to the global setting I see here is to make it only process 1 time per kill. This code is ran x amount of times of people in player. My recommendation is a localvar check on the mob in the global file or the isKiller, noKiller.

The way this is written it will be a problem on every assault that uses the global progressInstance()

Reply:
https://github.com/AirSkyBoat/AirSkyBoat/blob/3d8027430d88dd67514de46881f9d74346f335d2/scripts/globals/assault.lua#L221